category/tutorial/index.html

```diff
@@ -1,4 +1,4 @@
-    <!DOCTYPE html>
+<!DOCTYPE html>
```

Comment:
@shresth26 Looks like we have some indentation issue in the master branch
cc @ujwal-k